### PR TITLE
Style Note App

### PR DESCRIPTION
**Pull request by SanskrutiRP**
I have added CSS to the Notes App.
Below is the screenshot of the same.
As well as added many of the effects like hover,letter spacing, font and bordering.
And the home button which was deactivated that is also activated.
Kindly check this.
If any changes please inform me.

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
notes

### CODE REVIEW

**Review by urvashi-code1255 — COMMENTED** [review]
@SanskrutiRP Please resolve the mergee conflicts

**Comment by SanskrutiRP** [comment]
Will you please look into this?

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (991rajat) [inline .vscode/settings.json:2]: Don't push this file.

**Comment by 991rajat** [comment]
@SanskrutiRP  Please link the corresponding issue related to this PR.

**Pull request merged**